gravitee-apim-console-webui/src/components/gio-metadata/gio-metadata.component.ts

```diff
@@ -69,15 +69,15 @@ export class GioMetadataComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     this.referenceType = this.metadataSaveServices.type;
     this.permissionPrefix = this.referenceType === 'Global' ? 'environment' : this.referenceType.toLowerCase();
-    this.displayedColumns = ['global', 'key', 'name', 'format', 'value', 'defaultValue', 'actions'];
-
+    this.displayedColumns = ['global', 'key', 'name', 'format', 'value', 'actions'];
     this.metadataSaveServices
       .list()
       .pipe(
+        tap((metadata) => metadata['defaultValue'] && this.displayedColumns.push('defaultValue')),
         map((metadata) =>
           metadata.map((m) => ({
             name: m.name,
-            defaultValue: m.defaultValue,
+            defaultValue: m.defaultValue !== undefined ? m.defaultValue : undefined,
```

Comment:
Is it not exactly the same as what was there before?
```suggestion
            defaultValue: m.defaultValue
```

Reply:
Actually yes, fixed